include/control.h

```diff
@@ -52,6 +52,7 @@ class Config {
 public:
 	std::vector<std::string> startup_params = {};
 	std::vector<std::string> configfiles = {};
+	std::vector<std::string> configFilesCanonical = {};
```

Comment:
hmm.. if we put `std_fs::path` objects in the vector (instead of the string), it would eliminate the need to extract the `.string()` on lines 993 and 1003, making it a bit more direct to read.  Worth it?

Reply:
It works, so why not :)